ProjectLibertyLabs/gateway
# chore: various fixes to get e2e tests working for account service

# Problem

After changing to the mono-repo, e2e tests do not work because the scripts that execute them need updates, due to file locations changing.

# Solution

Update the e2e test workflow so that the tests can execute locally.

## Steps to Verify:

1. Follow the steps in `./developer-docs]account/README.md` and verify that the tests execute correctly.
